src/base/kaldi-error.cc

```diff
@@ -33,7 +33,11 @@
 
 #include "base/kaldi-common.h"
 #include "base/kaldi-error.h"
+
+// KALDI_GIT_HEAD is useless currently in full repo
```

Comment:
I'm a little skeptical of getting rid of KALDI_GIT_HEAD altogether. It shouldn't be hard to retain it. I feel like some downstream user of kaldi like @kkm000 (or whoever added this) might require it for some reason.